PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -440,7 +445,7 @@ private extension PanModalPresentationController {
 
         guard
             let scrollView = presentable?.panScrollable,
-            !scrollView.isScrolling
+            !scrollView.isScrolling && presentable?.shouldConfigureScrollViewInsets == true
```

Comment:
To keep the coding style consistent, I'd suggest
-  replacing the `&&` operator with a `,` like in the previous statement 
-  remove the `== true` as `shouldConfigureScrollViewInsets` is a boolean
-  giving this it's own line and not appending to the previous for better legibility